docs/topics/getting-started-cd-osl.adoc

```diff
@@ -0,0 +1,107 @@
+= Continuous Delivery using {OpenShiftLocal}
+
+In addition to building on your local machine and pushing to a local or public cloud, you also have the ability to set up continuous delivery for a booster. Specifically you can use the {launcher} application to clone a new booster repository to your GitHub account and have it deployed to your {OpenShiftLocal}. 
+
+. link:{link-launcher-openshift-local-install-guide}#create-launcher-app[Install the {launcher}] application to your {OpenShiftLocal}.
+. Navigate to the {launcher} application on your {OpenShiftLocal} using your browser. 
++
+[NOTE]
+====
+You can use the following command to open the Web Console of your {OpenShiftLocal} in your browser in order to navigate to your {launcher} application:
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ minishift console
+----
+====
+. Select _Prepare for Takeoff_.
+. Choose _Use {OpenShiftOnline} to build and deploy_.
+. Choose `REST API Level 0` and click _Next_.
+. Choose the {WildFlySwarm} runtime.
+. Enter an _OpenShift Project name_. You can also update any of the additional metadata fields or leave them as the default. Click _Next_.
+. Click the _Launch on OpenShift_ button.
++
+This will:
++
+** Create a new GitHub repository associated with your account containing the booster's code.
+** Create a pipeline to continuously deliver your booster's code.
+** The pipeline will then build and deploy your booster's code your {OpenShiftLocal}.
```

Comment:
The pipeline will then build and deploy your booster's code __to__ your {OpenShiftLocal}